Some more front matter maintenance (#816)
giantswarm/docs: src/content/ui-api/gsctl/cluster-definition.md

aliases:
  - /reference/cluster-definition/
user_questions:

>   - What are additional fields I need to fill out for node pool clusters?

Reviewer: (could also say "required")
```suggestion
  - What additional configuration options are available for node pool clusters?
```
